### PR TITLE
feat(catalog/[...]): schedule via config + backend-plugin-api support

### PR DESCRIPTION
**Pull request by pjungermann**
## Hey, I just made a Pull Request!

For: awsS3, azure, bitbucketServer, gerrit, gitlab

- Add option to configure schedule via `app-config.yaml`
- Add backend plugin

Relates-to: PR #13859

_(github, ldap, msgraph will follow in separate PRs.)_

#### :heavy_check_mark: Checklist

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by pjungermann** [comment]
remove stale

**Comment by pjungermann** [comment]
@freben just in case this one slipped through / was overlooked

**Comment by freben** [comment]
Yep we just piled up a bunch of stuff, on it!

File: plugins/catalog-backend-module-aws/config.d.ts [hunk 1]
```diff
@@ -45,22 +41,23 @@ export interface Config {
         | {
             /**
              * (Required) AWS S3 Bucket Name
-             * @visibility backend
```
Comment (freben) [inline plugins/catalog-backend-module-aws/config.d.ts:48]: can do the same removals in the Record below

**Review by freben — APPROVED** [review]
Whoa, solid stuff :)

File: plugins/catalog-backend-module-aws/config.d.ts (hunk 1 above)
Comment (freben) [reply plugins/catalog-backend-module-aws/config.d.ts:48]: ...some day perhaps :)

**Pull request merged**